styles.css

```diff
@@ -1,3 +1,7 @@
+.card-text .form-group>.col-md-9 .form-control {
```

Comment:
Can you please limit all of these css rules so they only apply inside the block

eg 

```
[data-block="enrolkey"] .card-text .form-group>.col-md-9 .form-control {
```